packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -31,7 +32,10 @@ export abstract class BaseResource {
       clerkMissingFapiClientInResources();
     }
 
-    const { payload, status, statusText } = await BaseResource.fapiClient.request<J>(requestInit);
+    const { payload, status, statusText } = await runWithExponentialBackOff(
+      () => BaseResource.fapiClient.request<J>(requestInit),
+      { maxRetries: 4, firstDelay: 800 },
```

Comment:
❓ How does the first delay option work? Does it affect the first retry?

Reply:
The first call happens instantly, if it fails, it retries after 800 ms (as `firstDelay` indicates), and then it continues with exponential logic that relates to `firstDelay`. Maybe `firstDelay` sounds a little confusing.